src/main/java/com/extremelyd1/gui/TeleportGui.java

```diff
@@ -0,0 +1,75 @@
+package com.extremelyd1.gui;
+
+import com.extremelyd1.game.Game;
+import com.extremelyd1.util.ArrayUtil;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+
+import java.util.Arrays;
+
+public class TeleportGui implements IGui{
```

Comment:
There is a space missing between the interface name and the bracket.

This happens a few more times in this pull request, perhaps try reformatting if you IDE supports this.